WooCommerce/WooCommerceTests/ViewRelated/Products/Add Product/AddProductCoordinatorTests.swift

```diff
@@ -41,7 +41,7 @@ final class AddProductCoordinatorTests: XCTestCase {
         assertThat(coordinator.navigationController.presentedViewController, isAnInstanceOf: BottomSheetViewController.self)
     }
 
-    func test_it_presents_AddProductWithAIActionSheet_on_start_when_eligible_for_ProductCreationAI_and_store_has_no_products() {
```

Comment:
👀 Should we also add a test to check that the old bottom sheet is displayed when the store is not eligible?

Reply:
Added in https://github.com/woocommerce/woocommerce-ios/pull/10747/commits/524eef4c965a5eea91dc71e9081039ed82c468d6